src/askscript/parser/askscript.grammar.pegjs

```diff
@@ -109,9 +108,7 @@ nonArithmExpression =
 
 brackets = '(' ws* v:value ws* ')' { return v }
 
-// === function definition ===
-
-functionDefinition = fS:functionSignature ws* '=' ws* fO:functionObject {                             return new ask.FunctionDefinition(fS, fO) }
```

Comment:
FYI @czerwinskilukasz1 variable definition can already express that - I removed this construct to make it easier to prettify the code based on AST

Reply:
OK